Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -25,17 +25,30 @@
 #import <Backpack/Font.h>
 #import <Backpack/Spacing.h>
 
-#import "BPKCalendar.h"
-
 @interface BPKCalendarCell ()
 
 @property(weak, nonatomic) CAShapeLayer *selectionLayer;
 @property(weak, nonatomic) CAShapeLayer *samedayLayer;
+@property(class, readonly) CGFloat titleHeight;
```

Comment:
Probably better to use C constants at the file level instead. This is somewhat wasteful as it will involve generating and calling class method via message dispatch.

So instead of `@property(class, readonly) CGFloat titleHeight;` + the method implementation use `const CGFloating BPKCalendarCellTitleHeight = 31.0;` at the top level.

[`BPKNavigationBar.m`](https://github.com/Skyscanner/backpack-ios/blob/master/Backpack/NavigationBar/Classes/BPKNavigationBar.m#L30) has an example.

Reply:
Thanks - that's much better!